### PR TITLE
Add a backend-adapters guide

### PR DESCRIPTION
**Pull request by mcasper**
We've gotten quite a few questions about how to use the SQLite adapater
(since all of our examples and guides are written using the Postgres
adapter), and quite a few questions when `infer_schema!` doesn't work on
SQLite. It would be nice to have a guide to point to, both in the early
days while a few things don't quite work with SQLite, and in the future
for the question "How do I use the X adapter?"

I'm the first to admit that I'm not fluent in the frontend ways, so if we decide this should see the light of day, it should get a makeover from someone better than I.

This turns the `Guides` button in the top right into a dropdown menu, with `Getting Started` and `Backend Adapters` links.

### CODE REVIEW

**Comment by sgrif** [comment]
Can you attach some screenshots of what the additions look like?

**Comment by mcasper** [comment]
screen shot 2016-03-08 at 8 25 39 am

Eww ^^

screen shot 2016-03-08 at 8 25 53 am

File: source/partials/_header.html.slim [hunk 1]
```diff
@@ -5,6 +5,10 @@ header.main-header
     nav.header-nav
       ul
         li.nav-item
-          a(href="/guides/getting-started") Guides
+          div.dropdown
+            a(href="#" onclick="toggleDropdown()" class="dropdown-btn") Guides
```
Comment (sgrif) [inline source/partials/_header.html.slim]: The href isn't needed.

**Comment by sgrif** [comment]
Would like to see more in depth details at some point, but this looks fine. Please verify that the dropdown is reasonable on mobile as well.

**Comment by sgrif** [comment]
@mcasper What's the status on this?

**Comment by mcasper** [comment]
Sorry for neglecting this for so long.

Just pushed up a tweak to make the dropdown reasonable for mobile, and I agree about the details - I'll keep adding info to this as we go along

File: source/partials/_header.html.slim (hunk 1 above)
Comment (mcasper) [inline source/partials/_header.html.slim]: True, we can make the link hover behave like a button with CSS.

**Pull request closed without merging**